package.json

```diff
@@ -19,28 +19,29 @@
   "dependencies": {
     "globby": "11.0.3",
     "gray-matter": "4.0.3",
-    "highlight.js": "11.3.1",
+    "highlight.js": "11.4.0",
     "js-paging": "0.1.2",
-    "marked": "3.0.4",
-    "next": "10.2.0",
+    "marked": "4.0.12",
+    "next": "12.0.10",
     "next-plugin-preact": "3.0.6",
-    "nodemailer": "6.7.0",
+    "nodemailer": "6.7.2",
     "nodemailer-mandrill-transport": "1.2.0",
-    "preact": "10.5.15",
+    "preact": "10.6.5",
     "preact-render-to-string": "5.1.19",
     "react": "npm:@preact/compat@17.0.3",
     "react-dom": "npm:@preact/compat@17.0.3",
-    "reading-time": "1.5.0"
+    "reading-time": "1.5.0",
+    "scheduler": "0.20.2"
```

Comment:
todo: remove in the future (now is mandatory to use preact + next.js, however it will be solved in the future)